staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3675,6 +3690,74 @@ func buildListener(port v1.ServicePort, annotations map[string]string, sslPorts
 	return listener, nil
 }
 
+func (c *Cloud) parseNLBHealthCheckAnnotations(portMapping *nlbPortMapping, annotations map[string]string) error {
+	getStringValue := func(annotation string, defaultVal string) string {
```

Comment:
humm, maybe better to define some function to make it reusable? similar to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/legacy-cloud-providers/aws/aws.go#L3562(however, ideally that should be something like `ParseStringListAnnotation`)

Reply:
Personally i favor interface like below
```
func parseStringAnnotation(annotations map[string]string, annotation string, value *string) bool {
}
func parseInt64Annotation(annotations map[string]string, annotation string, value *int64) bool, error {
}
```
so to use it with default value, you just need to things like 
```
var a int64 = 100
exists, error := parseInt64Annotation(annotations, "some-annotation", &a)
```

Reply:
thanks for the recommendation.